pkg/noderesourcetopology/filter_test.go

```diff
@@ -621,159 +639,155 @@ func TestNodeResourceTopologyMultiContainerContainerScope(t *testing.T) {
-						v1.ResourceCPU:    resource.MustParse("32"),
-						v1.ResourceMemory: resource.MustParse("16Gi"),
-					},
-				},
-				[]v1.ResourceList{
-					{
-						v1.ResourceCPU:    resource.MustParse("16"),
-						v1.ResourceMemory: resource.MustParse("16Gi"),
-					},
-				},
-			),
-			node: nodes[0],
-			nrts: []*topologyv1alpha1.NodeResourceTopology{
-				nodeTopologies[0],
-			},
-			avail:      []resourceDescriptor{},
-			wantStatus: nil,
+			description: "[4][tier2] multi-containers with good allocation, spread across NUMAs - fit",
+			cntReq:      []map[string]string{{cpu: "20", memory: "40G"}, {cpu: "20", memory: "40G"}},
+		},
+		{
+			description: "[4][tier1] multi containers with good devices and hugepages allocation, spread across NUMAs - fit",
+			cntReq:      []map[string]string{{cpu: "2", memory: "6G", hugepages2Mi: "500Mi", nicResourceName: "16"}, {cpu: "2", memory: "6G", hugepages2Mi: "50Mi", nicResourceName: "8"}},
+		},
+		{
+			description: "[5][tier2] multi-containers with cpu over allocation, spread across NUMAs - not fit",
+			cntReq:      []map[string]string{{cpu: "20", memory: "40G"}, {cpu: "20", memory: "40G"}, {cpu: "20", memory: "1G"}},
+			statusErr:   "cannot align container: cnt-3",
+		},
+		{
+			description: "[5][tier2] multi-containers with memory over allocation, spread across NUMAs - not fit",
+			cntReq:      []map[string]string{{cpu: "20", memory: "40G"}, {cpu: "20", memory: "40G"}, {cpu: "2", memory: "40G"}},
+			statusErr:   "cannot align container: cnt-3",
+		},
+		{
+			description: "[5][tier1] multi containers with over hugepages allocation, spread across NUMAs - not fit",
+			cntReq:      []map[string]string{{cpu: "2", memory: "6G", hugepages2Mi: "500Mi"}, {cpu: "2", memory: "6G", hugepages2Mi: "500Mi"}},
+			statusErr:   "cannot align container: cnt-2",
+		},
+		{
+			description: "[5][tier1] multi containers with over devices allocation, spread across NUMAs - not fit",
+			cntReq:      []map[string]string{{cpu: "2", memory: "6G", hugepages2Mi: "50Mi", nicResourceName: "20"}, {cpu: "2", memory: "6G", hugepages2Mi: "500Mi", nicResourceName: "20"}},
+			statusErr:   "cannot align container: cnt-2",
+		},
+		{
+			description: "[7][tier1] init container with cpu over allocation, multi-containers with good allocation - not fit",
+			initCntReq:  []map[string]string{{cpu: "40", memory: "40G"}},
+			cntReq:      []map[string]string{{cpu: "1", memory: "4G"}, {cpu: "1", memory: "4G"}},
+			statusErr:   "cannot align init container: cnt-1",
+		},
+		{
+			description: "[7][tier1] init container with memory over allocation, multi-containers with good allocation - not fit",
+			initCntReq:  []map[string]string{{cpu: "4", memory: "70G"}},
+			cntReq:      []map[string]string{{cpu: "1", memory: "4G"}, {cpu: "1", memory: "4G"}},
+			statusErr:   "cannot align init container: cnt-1",
+		},
+		{
+			description: "[7][tier1] init container with memory over allocation, multi-containers with good allocation - not fit",
```

Comment:
this looks the same as the previous one. is it intentional?

Reply:
Copy-paste mistake probably.